src/sqlite-async.ts

```diff
@@ -210,17 +223,20 @@ export class Database {
 // The Statement class
 //-----------------------------------------------------------------------------
 
-class Statement {
-  constructor(statement) {
+export class Statement {
+
+  statement: sqlite.Statement
+
+  constructor(statement: sqlite.Statement) {
     if (!(statement instanceof sqlite.Statement)) {
       throw new TypeError(`Statement: 'statement' is not a statement instance`);
     }
     this.statement = statement;
   }
 
-  bind(...args) {
+  bind(...args: any[]) {
     return new Promise((resolve, reject) => {
-      let callback = (err) => {
+      const callback = (err: Error|null) => {
```

Comment:
I switched to `const` on lines that were changed because of types. Not strictly necessary.